cfme/cloud/availability_zone.py

```diff
@@ -18,6 +20,14 @@
 match_page = partial(match_location, controller='availability_zone', title='Availability Zones')
 
 
+class CloudAvailabilityZoneTimelinesView(TimelinesView, BaseLoggedInPage):
+    @property
+    def is_displayed(self):
+        return self.logged_in_as_current_user and \
+            self.navigation.currently_selected == ['Compute', 'Clouds', 'Availability Zones'] and \
+            super(TimelinesView, self).is_displayed
```

Comment:
Just a note, I've been thinking of using this syntax for the 'logged_in_as_current_user - move that up to BaseLoggedInPage.is_displayed and just call it from the child views. </thought>